postgres_kernel/kernel.py

```diff
@@ -181,16 +181,24 @@ def do_execute(self, code, silent, store_history=True,
                     'ename': 'ProgrammingError', 'evalue': str(e),
                     'traceback': []}
         else:
-            if header is not None:
+            # Modified by mhoangvslev
+            self.send_response(
+                    self.iopub_socket, 'stream', {
+                        'name': 'stdout',
+                        'text': str(len(rows)) + " row(s) returned."
+                        })
```

Comment:
This change is ensuring we always have output, even if there's no rows in the result?

The indentation is a little funky. Do you think you could change it to use 4 spaces consistently?

Reply:
The comments I put in is just to tell people what have been modified without making them look up the commit history. I will remove it if you so wish, nonetheless.